base/tools/src/main/java/com/netscape/cmstools/profile/ProfileFindCLI.java

```diff
@@ -55,11 +67,15 @@ public void execute(CommandLine cmd) throws Exception {
         s = cmd.getOptionValue("size");
         Integer size = s == null ? null : Integer.valueOf(s);
 
+        Boolean visible = cmd.hasOption("visible") ? Boolean.TRUE : null;
+        Boolean enable = cmd.hasOption("enable") ? Boolean.TRUE : null;
```

Comment:
This code doesn't allow us to find hidden / disabled profiles. How about something like this?
```
Boolean visible = cmd.hasOption("visible") ? Boolean.valueOf(cmd.getOptionValue("visible")) : null;
Boolean enable = cmd.hasOption("enable") ? Boolean.valueOf(cmd.getOptionValue("enable")) : null;
```

Reply:
In this way the option will require the explicit value `true` or `false` and for a boolean I do not like. However,  this is a particular case because it allows  to manage all 3 possible conditions: all, only true and only false. Therefore I have update accordingly.

Reply:
Thanks for the update! The alternative is to provide options like: `--visible-only`, `--hidden-only`, `--enabled-only`, `--disabled-only`. I'll let you decide. Regardless, there's no objection from me.

If these changes are sufficient for IPA feel free to merge. Thanks!